pkg/synthetics/synthetics_api_integration_test.go

```diff
@@ -1022,3 +1024,345 @@ func newIntegrationTestClient(t *testing.T) Synthetics {
 
 	return New(tc)
 }
+
+// TestSyntheticsStartAutomatedTest_Basic performs a test by creating three monitors and using the
+// syntheticsStartAutomatedTest mutation to create a batch with these three monitors. The expected
+// behaviour of this test is to return a valid batchId and throw no error.
+func TestSyntheticsStartAutomatedTest_Basic(t *testing.T) {
+	t.Parallel()
+	t.Skipf(
+		`Temporarily skipping tests associated with the Synthetics Automated Tests feature, ` +
```

Comment:
Good to proceed for now, hoping we shall take a note of re-testing once API is stable.

Reply:
Right - these tests are currently being skipped, given that the API is not producing expected results (fully) on production yet, and is a bit more stable on staging, but is still erroneous at times, leading to test failures of some of these, such as the unit tests written. However, all of these tests have been locally verified, and have been written in alignment with the expected behaviour of the API, to ensure these tests can be unskipped once the API is stable.